x-pack/test/api_integration/apis/maps/migrations.js

```diff
@@ -76,7 +76,7 @@ export default function ({ getService }) {
         }
         expect(panels.length).to.be(1);
         expect(panels[0].type).to.be('map');
-        expect(panels[0].version).to.be('7.14.0');
+        expect(panels[0].version).to.be('7.15.0');
```

Comment:
This change is about this [failure](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/143831/execution/node/714/log): I don't understand why the version changes from 7.14 to 7.15 for map saved objects if this PR only migrates lens SO. Can someone help me understand? 🤔

Reply:
I think that could be a side-effect of how dashboard panel migrations work. It could be that as soon as "any" migration for 7.15.0 for embeddables exists, all panels will run through migration and get a matching migration applied, but the version will be set to that version, whether it had a migration for that specific panel or not. Meaning panels containing map embeddables would now have 7.15.0 on them, since they theoretically have seen "all 7.15.0 migrations" (even though "all 7.15.0 migrations" did not contain any migration for Maps). @ThomThomson is my understanding here correct?

I also agree that **if** that is correct this is a bit unnice noise in the testing, that we cause map failures when adding a Lens embeddable migration, and would wonder again, if there is any better way we can maybe test this? (Question also directed to Devon :D)